sections/k-partners-listing.liquid

```diff
@@ -0,0 +1,467 @@
+                                                            assign grid_space_tablet = number_of_columns_mobile | minus: 1 | times: settings.spacing_grid_horizontal | plus: 100 | append: 'px'
+                                                            assign grid_space_mobile = number_of_columns_mobile | minus: 1 | times: settings.spacing_grid_horizontal | divided_by: 2 | plus: 30 | append: 'px'
+                                                            assign image_width = 1
+                                                        -%}
+                                                        {% capture sizes %}
+                                                            (min-width: {{ settings.page_width }}px) calc(({{ settings.page_width }}px - {{ grid_space_desktop }}) * {{ image_width }} /  {{ number_of_columns }}),
+                                                            (min-width: 990px) calc((100vw - {{ grid_space_desktop }}) * {{ image_width }} / {{ number_of_columns }}),
+                                                            (min-width: 750px) calc((100vw - {{ grid_space_tablet }}) * {{ image_width }} / {{ number_of_columns_mobile }}),
+                                                            calc((100vw - {{ grid_space_mobile }}) * {{ image_width }} / {{ number_of_columns_mobile }})
+                                                        {% endcapture %}
+                                                        {{
+                                                            object.logo
+                                                            | image_url: width: 300
+                                                            | image_tag:
+                                                            widths: '50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3200',
+                                                            sizes: sizes
+                                                        }}
+                                                    </div>
+                                                {% else %}
+                                                    <div class="placeholder-tab-img">
+                                                        {% render 'icon-setting' %}
+                                                    </div>
+                                                {% endif %}
+                                            </div>
+                                            <div class="t-flex-column">
+                                                <h3 class="title inline-richtext h2">{{ name }}</h3>
+                                                {% if object.city_region_service_area != blank %}
+                                                    <div class="description-column rte">
+                                                        <p>{{ object.city_region_service_area }}, {{ page_country }}</p>
```

Comment:
just show the text for `city_region_service_area` here. No need to append the country/state.